compiler/src/dmd/typesem.d

```diff
@@ -317,8 +317,13 @@ private void resolveHelper(TypeQualified mt, const ref Loc loc, Scope* sc, Dsymb
             {
                 if (id.dyncast() == DYNCAST.dsymbol)
                 {
-                    // searchX already handles errors for template instances
```

Comment:
This raises some questions about searchX here. Is there something that function isn't doing?

Reply:
isnt doing "anymore" ? yes. See the diff of #13946.